ocaml/testsuite/tests/translprim/array_spec.heap.flat.reference

```diff
@@ -23,7 +23,7 @@
       (let
         (eta_gen_len =
            (function {nlocal = 0} prim[genarray] stub
-             ignore assert all zero_alloc : int (array.length[gen] prim))
```

Comment:
This is some tests that I messed up earlier but didn't fix (because this is only tested when `--disable-stack-allocation`). After promotion, this PR no longer changes this test.